### ISSUE TITLE
Provide user the granuality to select which events to import from the contacts.

### ISSUE THREAD

**Issue by Addy-21** [m-i-n-a-r/birday#262]
Current import overwrites everything in the database, and this created multiple copies of the same event, if I had manually added an event in birday app and the said event also exists in the contacts info.

For example, I have created an event for my parent's marriage anniversary in the birday app manually, and my father's Google contact also contains the anniversary date. So, when I import events from the contacts app, it created a new anniversary instance and now I have two anniversary events- one, that I created manually inside the app, and a second, from the Google contacts app.

Now, you might say to remove my custom event and use the Google one, but in the custom one, I use a different image(of both my father and mum) and title (eg.- ma-pa's anniversary) whereas the Google one will show my dad's image and title is "Dad" anniversary.

With the option of ignoring the import of my dad's anniversary info, I can keep using the custom one without any copies.

**Comment by Addy-21** [comment]
Yeah, with auto sync turned off and doing manual import, you can add the
feature to choose. It would even be better if shows a comparison b/w the
choice and the one already in the database.

**Comment by m-i-n-a-r** [comment]
That's pretty complex to develop properly, but I can do this! I'll update the issue as soon as i start to develop this feature!

**Comment by m-i-n-a-r** [comment]
Done in 9778a3ff64ea294eef18764ec859312ef4fc5cd7 finally :) and available in the next release!
